jadx-core/src/main/java/jadx/core/xmlgen/ManifestAttributes.java

```diff
@@ -164,6 +164,9 @@ private void parseValues(String name, NodeList nodeList) {
 	public String decode(String attrName, long value) {
 		MAttr attr = attrMap.get(attrName);
 		if (attr == null) {
+			if (attrName.contains(":")) {
+				attrName = attrName.split(":")[1];
```

Comment:
I would use `attrName.split(":",2)[1];` 

To my knowledge the attribute name after the namespace definition can not contain colons so your code wouldn't make problems, so again this change would only improve execution performance.

Reply:
For serach we use full or short name attr like this code
https://github.com/skylot/jadx/blob/ea2d54c36e6ea806d968ab7e89079b16198e6971/jadx-core/src/main/java/jadx/core/xmlgen/BinaryXMLParser.java#L304-L305

 >To my knowledge the attribute name after the namespace definition can not contain colons
 
App package attributes might contains a `app:someArrt`  or `someAttrt` so me like `package` in AndroidManifest, so this check is needed to avoid an IndexOutOfBoundsException.

Reply:
Sorry the check was out of scope, my comment was just only about the split call.